.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
node_modules should really be in a global .gitignore not a project one, other two are good though

Reply:
@calvinmetcalf Some projects check in node_modules, I haven't typically seen people use global gitignores like that

Reply:
checking in a node_modules is an exceptional thing, not a typical thing so that doesn't belong here (as opposed to the  package and yarn lock since a given dev will need to check some in and not others), you're generally not going to see global gitignores since those aren't going to be  included in projects.

Reply:
@calvinmetcalf Okay, it doesn't really bother me to take it out - every gitignore I've ever seen across probably thousands of projects I've worked on includes this but I will remove it.